docs/faq/templates.md

```diff
@@ -1,15 +1,15 @@
 ??? info "What are nuclei templates?"
 
-	Nuclei [templates](http://github.com/projectdiscovery/nuclei-templates) are the core of the nuclei project. The template contains the actual logic to execute for detecting various vulnerabilities and checks, template projects consist ==880==+ ready-to-use **[community-contributed](https://github.com/projectdiscovery/nuclei-templates/graphs/contributors)** vulnerability templates.
+	Nuclei [templates](http://github.com/projectdiscovery/nuclei-templates) are the core of the nuclei project. The template contains the actual logic to execute for detecting various vulnerabilities and checks. The template project consists of **several thousand** ready-to-use **[community-contributed](https://github.com/projectdiscovery/nuclei-templates/graphs/contributors)** vulnerability templates.
```

Comment:
We could loose the "template" word from the "The template project consists"...it's repetitive, and should already be clear from the context